test/paymaster.test.ts

```diff
@@ -95,9 +95,10 @@ describe('EntryPoint with paymaster', function () {
         calldata = await account.populateTransaction.execute(account.address, 0, updateEntryPoint).then(tx => tx.data!)
       })
       it('paymaster should reject if account doesn\'t have tokens', async () => {
-        const op = await fillAndSign({
+        const op = await fillSignAndPack({
           sender: account.address,
-          paymasterAndData: paymaster.address,
+          paymaster: paymaster.address,
+          paymasterPostOpGasLimit: 3e5,
```

Comment:
just like other fields, fillAndSign should have default value for postOpGasLimit.

Reply:
this is why I think we should add postOpGasLimit to the defaults: to avoid adding it in every test below.

Reply:
I don't think that postOp should have a default non-zero value